keras_cv/layers/preprocessing/random_hue_test.py

```diff
@@ -97,12 +97,3 @@ def test_with_unit8(self):
         layer = preprocessing.RandomHue(factor=(0.3, 0.8))
         output = layer(image)
         self.assertNotAllClose(image, output)
-
-    def test_config(self):
```

Comment:
Same here, any reason the tests are removed?

Reply:
Sorry should've left a comment: I feel this is covered in serialization test.  Do you agree with this?  If not, can revert

Reply:
Updated - removed the second half though as this seems unneeded as it just tests a helper method in preprocessing utils really.